Readme.md

```diff
@@ -1,33 +1,24 @@
 <!-- default file list -->
 *Files to look at*:
 
-* [Form1.cs](./CS/LocalizableDisplayName/Form1.cs) (VB: [Form1.vb](./VB/LocalizableDisplayName/Form1.vb))
+* [LocalizableDisplayNameAttribute.cs](./CS/LocalizableDisplayName/LocalizableDisplayNameAttribute.cs) (VB: [LocalizableDisplayNameAttribute.vb](./VB/LocalizableDisplayName/LocalizableDisplayNameAttribute.vb))
 * [Product.cs](./CS/LocalizableDisplayName/Product.cs) (VB: [Product.vb](./VB/LocalizableDisplayName/Product.vb))
-* [Program.cs](./CS/LocalizableDisplayName/Program.cs) (VB: [Program.vb](./VB/LocalizableDisplayName/Program.vb))
-* [PropertyNamesRes.de.resx](./CS/LocalizableDisplayName/PropertyNamesRes.de.resx) (VB: [PropertyNamesRes.de.resx](./VB/LocalizableDisplayName/PropertyNamesRes.de.resx))
-* [PropertyNamesRes.resx](./CS/LocalizableDisplayName/PropertyNamesRes.resx) (VB: [PropertyNamesRes.resx](./VB/LocalizableDisplayName/PropertyNamesRes.resx))
 <!-- default file list end -->
-# How to localize displayable names of persistent properties
+# How to localize the displayable name of a property
 
+.NET Framework has a built-in mechanism to customize a displayable name. Refer to the [DisplayNameAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.displaynameattribute) description for details.
 
-<p>This example demonstrates how to apply DXDisplayName to persistent properties. It's also required to add neutral and localized resource files to the project and set the DXDisplayNameAttribute.UseResourceManager static property to True:</p>
-
-```cs
-<newline/>
-System.ComponentModel.DXDisplayNameAttribute.UseResourceManager = true;<newline/>
-
-```
+You can create a DisplayNameAttribute descendant and override its [DisplayName](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.displaynameattribute.displayname) property. In the DisplayName property getter, retrieve a corresponding value from a resource file. Refer to the following MSDN article for additional information: [Work with .resx files programmatically](https://docs.microsoft.com/en-us/dotnet/framework/resources/working-with-resx-files-programmatically).
```

Comment:
`retrieve a corresponding value` There can be multiple corresponding values?

Reply:
I deleted the second sentence and make the first sentence more specific: "You can create a DisplayNameAttribute descendant and override its [DisplayName](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.displaynameattribute.displayname) property **to return a value depending on the user language**."

Reply:
`depending on the user language`
- What is this 'user language' you mention?
- Why do you use 'depending' instead of 'that depends on'?

Reply:
1. I discussed this with teammates and we decided that "system locale" is a better technical term than "user language" in this context. I changed "user language" to "system locale".
2. Yes, I need to change "depending" to "that depends on". Done.